backend/config/config.go

```diff
@@ -315,3 +316,7 @@ func (s *Session) Validate() error {
 	}
 	return nil
 }
+
+type AuditLog struct {
+	EnableStoring bool `yaml:"enable_storing" json:"enable_storing" koanf:"enable_storing"`
```

Comment:
👍

Reply:
Probably should also be documented in https://github.com/teamhanko/hanko/blob/add-audit-logs/backend/docs/Config.md

Reply:
I can rename it to `enabled`, but one thing to consider is, that currently the `audit_log.Client` only stores the logs in the database but it should also log it to `stdout`, which is not implemented yet. If we rename the filed to `enabled` and is set to false, will the logging to `stdout` also be disabled? Normally you don't want to disable the audit logging to `stdout`.

Maybe we should do it something like that: `audit_log.storage.enabled` 
And if we support different storage (e.g. different database, ...) in the future it would be easier to add it.

Reply:
Hmm why do we want to log the audit log to `stdout`?

Reply:
When you host hanko yourself, maybe you already have a log aggregation system like (e.g. ELK-stack, Graylog, ...) and do your audit log review there then you don't want to build a new system that uses the private API to get the audit logs.

Reply:
https://github.com/teamhanko/hanko/pull/185/commits/cc5504e24eb4a450bd3eccf7e5d3ea9346759b17 Now it logs to stderr. What if someone don't want that? I can turn off the the main audit log feature via `audit_log.storage.enabled`, but can't turn off the log to stderr. Also wouldn't it make sense to control the filehandle, log format, etc? And can we confirm it works with e.g. ELK or Graylog? I didn't tested it.

Reply:
@FreddyDevelop any news?

Reply:
Yes, you are right with you suggestions above. I will add a config option that you can change the output to stderr or stdout.

It would make sense, to control the log format, but i think we can do it later, because we should change the log format of all logs and not only the audit log ones.

Reply:
Ok, it's better now, but I still can't enable `audit_log.storage` while the output to stderr or stdout is disabled. The audit log contains sensitive data that you may not want to output via stderr or stdout at all.

And I don't think it makes sense to omit sensitive data via the introduced `LogSensitiveValues` option. What can I do with the censored audit log? The application log contains almost the same information.

Reply:
So basically you just want two config options, one to enabled/disable audit logs output to stdout/stderr and one that enables/disables storage of the audit logs to the DB. And the two options don't depend on each other. Is that correct?

Reply:
Yes, I think this would be a good approach.